extra-python/pyxtrlock/autobuild/prepare

```diff
@@ -0,0 +1,3 @@
+abinfo "Adding the code of utf-8 into setup.py"
```

Comment:
I don't think this `abinfo` is sufficiently accurate for what this does. Where did this routine come from (from you or from another distribution? If it's the latter, please cite this line in `abinfo`).